build/arch.mips.mk

```diff
@@ -24,4 +24,9 @@ endif
 # Added to all files
 GCC_ABIFLAGS += -msoft-float
 CLANG_ABIFLAGS += -msoft-float
+ifeq ($(KGPROF), 1)
+  CFLAGS_KGPROF = -finstrument-functions \
+                  -finstrument-functions-exclude-file-list=lib,bin,sys/mips,spinlock.c  \
```

Comment:
Why do you exclude lib and bin?
In my opinion that flag should be used only for kernel.